config/examples/nnf_v1alpha1_nnfdatamovementprofile.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfDataMovementProfile
+metadata:
+  name: default
+  namespace: nnf-system
+data:
+  default: true
+  slots: 8
+  maxSlots: 0
+  command: ulimit -n 2048 && mpirun --allow-run-as-root --hostfile $HOSTFILE dcp --progress 1 --uid $UID --gid $GID $SRC $DEST
+  logStdout: false
+  storeStdout: false
+  progressIntervalSeconds: 5
+---
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfDataMovementProfile
+metadata:
+  name: no-xattr
```

Comment:
Is the no-xattr profile specifically for tests, or do you consider this to be a production profile?

Reply:
It's used only for copy_out to global lustre, since our global lustre servers do not support extended attributes. copy_in doesn't need this.

Reply:
Then this is site-specific, so it should live only in the gitops repo, not here in the nnf-sos repo.